eldritch/items/__init__.py

```diff
@@ -0,0 +1,70 @@
+from eldritch.assets import Deputy
+from .base import Item, Weapon, OneshotWeapon, Tome
+from .common import (
+    AncientTome, Automatic45, Axe, DarkCloak,
+    Derringer18, Revolver38, Dynamite,
+    TommyGun, Food, ResearchMaterials,
+    Rifle, Shotgun, CavalrySaber, Whiskey, Knife, Lantern,
+    Bullwhip, Cross)
+
+from .unique import (
+    HolyWater, EnchantedBlade, EnchantedKnife, MagicLamp, MagicPowder, SwordOfGlory
+)
+from .spells import (BindMonster, DreadCurse, EnchantWeapon,
```

Comment:
Food for thought: do you want to include skills/allies in the items submodule? I could go either way. On the one hand, having a single pattern for defining these cards is useful. On the other hand, `items` would become a misnomer, and `player_acquirable_cards` doesn't exactly roll off the tongue.

Reply:
I think you could call them `trade_items` like they do [in the wiki](http://www.arkhamhorrorwiki.com/Trade_items), but also ["item"](http://www.arkhamhorrorwiki.com/Item) seems clear enough.